src/SFA.DAS.EAS.Web/Views/Shared/_WhatIsEnglishPercentage.cshtml

```diff
@@ -1,7 +1,11 @@
-<details id="inform-english-percentage">
-    <summary><span class="summary">What is English percentage?</span></summary>
-    <div class="panel">
-        <p>Your English percentage is provided by HMRC based on the data in your payroll submissions. It represents the proportion of your payroll that's paid to your employees that live in England.</p>
-        <p>If you think there are any issues with your English percentage, check your payroll data to make sure your employees' address information is correct.</p>
+<details id="inform-english-percentage" class="govuk-details">
+    <summary class="govuk-details__summary">
+        <span class="govuk-details__summary-text">
+            What is English percentage? 
+        </span>
+    </summary>
+    <div class="govuk-details__text">
+        Your English percentage is provided by <abbr>HMRC</abbr> based on the data in your payroll submissions. It represents the proportion of your payroll that's paid to your employees that live in England.<br>
```

Comment:
Should this text be wrapped in a `<p>`?

Reply:
The GDS pattern shows raw text in the DIV. (Elsewhere, if text is split into two paragraphs I added 2x Ps; along the same principle of text within table cells.